src/test/java/com/epam/tc/hw3/pages/PageObject.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.tc.hw3.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class PageObject {
+
```

Comment:
since it's a base class it's better to make it abstract

Reply:
Done